### PR TITLE
Improve instructions for Scala 3

### PR DESCRIPTION
**Pull request by yatesco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # http4s giter8 template
 
-Generate an http4s service on the blaze backend with Circe.
+Generate an http4s (Scala 2) service on the blaze backend with Circe.
 
 1. [Install sbt](http://www.scala-sbt.org/1.0/docs/Setup.html)
 2. `sbt new http4s/http4s.g8`
```
Comment (armanbilge) [inline README.md]: What if we just do like this?
```suggestion
2. Scala 2: `sbt new http4s/http4s.g8`
   Scala 3: `sbt new http4s/http4s.g8 --branch 0.23-scala3`
```

**Comment by yatesco** [comment]
Better. I’ll update tomorrow and send it through.

File: README.md [hunk 2]
```diff
@@ -3,10 +3,11 @@
 Generate an http4s service on the blaze backend with Circe.
 
 1. [Install sbt](http://www.scala-sbt.org/1.0/docs/Setup.html)
-2. `sbt new http4s/http4s.g8`
+2. Create your project
+   1. Scala 2: `sbt new http4s/http4s.g8` 
+   2. Scala 3: `sbt new http4s/http4s.g8 --branch 0.23-scala3`
```
Comment (armanbilge) [inline README.md]:
```suggestion
2. Create your project:
   - Scala 2: `sbt new http4s/http4s.g8` 
   - Scala 3: `sbt new http4s/http4s.g8 --branch 0.23-scala3`
```

**Review by armanbilge — APPROVED** [review]
Thanks!

**Review by rossabaker — APPROVED** [review]

**Pull request merged**